target_chains/ethereum/contracts/contracts/pyth/PythGovernance.sol

```diff
@@ -190,4 +201,44 @@ abstract contract PythGovernance is
 
         emit ValidPeriodSet(oldValidPeriod, validTimePeriodSeconds());
     }
+
+    function setWormholeAddress(
+        SetWormholeAddressPayload memory payload,
+        bytes memory encodedVM
+    ) internal {
```

Comment:
should we restrict this to `private`

Reply:
good point, i followed the pattern in this file but i don't see any reason why not